opensubtitles/common.go

```diff
@@ -0,0 +1,143 @@
+package opensubtitles
+
+import "time"
+
+type Subtitles struct {
+	TotalPages int    `json:"total_pages"`
+	TotalCount int    `json:"total_count"`
+	Page       int    `json:"page"`
+	Data       []Data `json:"data"`
+}
+type Uploader struct {
+	ID   int    `json:"id"`
+	Name string `json:"name"`
+	Rank string `json:"rank"`
+}
+type FeatureDetails struct {
+	FeatureID   int    `json:"feature_id"`
+	FeatureType string `json:"feature_type"`
+	Year        int    `json:"year"`
+	Title       string `json:"title"`
+	MovieName   string `json:"movie_name"`
+	ImdbID      int    `json:"imdb_id"`
+	TmdbID      int    `json:"tmdb_id"`
+}
+type RelatedLinks struct {
+	Label  string `json:"label"`
+	URL    string `json:"url"`
+	ImgURL string `json:"img_url"`
+}
+type Files struct {
```

Comment:
Codacy found an issue: [exported type Files should have comment or be unexported](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)